### PR TITLE
Update lora's param_partition_spec.

### PR DESCRIPTION
**Pull request by JianyuWangV**

### CODE REVIEW

**Review by markblee — APPROVED** [review]

**Comment by madrob** [comment]
@JianyuWangV You may need to rebase your changes to get CI to run. Thanks!

**Comment by ruomingp** [comment]
Do we still need this PR?

**Comment by JianyuWangV** [comment]
> Do we still need this PR?

Yes, thanks for the reminder! Let me rebase the PR and test it.

File: axlearn/common/lora.py [hunk 1]
```diff
@@ -130,7 +129,6 @@ def __init__(self, cfg: _BaseLoraAdapter.Config, *, parent: Module):
             cfg.lora_up.set(
                 input_dim=cfg.rank,
                 output_dim=cfg.output_dim,
-                param_partition_spec=["model", None],
```
Comment (ruomingp) [inline axlearn/common/lora.py:133]: Is there a test for this change?

Comment (JianyuWangV) [reply axlearn/common/lora.py:133]: Added a test. PTAL.

**Review by ruomingp — APPROVED** [review]
Thanks!

**Review by ruomingp — APPROVED** [review]

**Pull request merged**